app/code/Magento/Elasticsearch/Model/Adapter/Elasticsearch.php

```diff
@@ -145,15 +145,19 @@ public function ping()
      *
      * @param array $documentData
      * @param int $storeId
+     * @param string $mappedIndexerId
      * @return array
      */
-    public function prepareDocsPerStore(array $documentData, $storeId)
+    public function prepareDocsPerStore(array $documentData, $storeId, $mappedIndexerId)
```

Comment:
Unfortunately, we are not allowed to change public methods signature according to the [Backward Compatible Development Guide](https://devdocs.magento.com/guides/v2.3/contributor-guide/backward-compatible-development#adding-parameters-in-public-methods).